src/components/legend/legend-item-toolbar/info-button/info-button-component.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Tooltip from 'vizzuality-components/dist/tooltip';
```

Comment:
I left this as it was because `rc-tooltip` provides an easy way to set the position of the tooltip like top-left / top-right. `react-tooltips` didn't want to work with me that easily and I didn't want to spend too much time playing with tooltip arrow. Nevertheless, it would be nice to have one library for tooltips in the project for sure :+1:  
but maybe not in this PR (please :D )
![avom7-dh5xb](https://user-images.githubusercontent.com/15097138/62145901-2e47e380-b2ec-11e9-82f9-de0f093217ba.gif)